baseapp/abci.go

```diff
@@ -242,6 +251,21 @@ func (app *BaseApp) Commit() (res abci.ResponseCommit) {
 	}
 }
 
+func (app *BaseApp) halt() {
+	app.logger.Info("halting node per configuration", "height", app.haltHeight, "time", app.haltTime)
```

Comment:
this will log a `0` if one is halt param is defined and the other not

Reply:
correct. This is fine imho.

Reply:
what if you call `halt` directly on each `switch` case? 

```go
func (app *BaseApp) halt(haltHeight, haltTime uint64) 
```
```go
// on `Commit()`
switch {
		case app.haltHeight > 0 && uint64(header.Height) == app.haltHeight:
			app.halt(app.haltHeight, 0)

		case app.haltTime > 0 && header.Time.UnixNano() >= int64(app.haltTime):
			app.halt(0, app.haltTime)
		}
}
```

Reply:
This doesn't improve brevity imho. The log spit out just simply states the operators config. I can see it being helpful to see both values in case things get wonky (i.e. when you supply both configs).